src/index.js

```diff
@@ -239,7 +239,7 @@ class RuntimeMetrics {
   _cpuHeap() {
     RuntimeMetrics.measureCpuHeap(this);
     const reg = this.registry;
-    this._intervals.push(reg.schedulePeriodically(RuntimeMetrics.measureCpuHeap, 10000, this));
+    this._intervals.push(reg.schedulePeriodically(RuntimeMetrics.measureCpuHeap, 60000, this));
```

Comment:
Noting for posterity, the symptoms we saw was an appearance of increased CPU usage for `nodejs.cpuUsage` because the max of the 10s samples would get used if an app had some 10s windows with higher usage. Averaging over the minute helps smooth that out when not looking at a higher resolution source.